components/Header/Header.module.scss

```diff
@@ -31,22 +37,92 @@
   }
 }
 
+.containerHeader {
+  display: flex;
+  flex-direction: column;
+}
+
+.containerNav {
+  display: flex;
+  padding: 0 24px;
+  justify-content: space-between;
+}
+
+$color1: #8DC8E9;
+$color2: #8C5AE3;
+
+@mixin linear-gradient($direction, $colors...) {
```

Comment:
Mixins make sense for "reusablitiy". If you are planing to use this mixin somewhere else in the future code, put it a single file and import. If not - additional entity is not required.